src/app/preferences/general-preferences/general-preferences.module.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference path="general-preferences.controller.ts"/>
+/// <reference path="general-preferences.service.ts"/>
+
+namespace Core {
+
+    export const generalPreferencesModule = angular
+      .module('hawtio-general-preferences', [])
+      .controller('PreferencesGeneralController', GeneralPreferencesController)
```

Comment:
`'PreferencesGeneralController'` -> `'GeneralPreferencesController'`
Don't change the controller name without reasons.